base/applications/network/netstat/netstat.c

```diff
@@ -119,16 +124,17 @@ BOOL ParseCmdline(int argc, wchar_t* argv[])
                     case L's':
                         bDoShowProtoStats = TRUE;
                         break;
-                    case L'o':
-                        bDoShowProcessId = TRUE;
+                    case L't':
+                        // UNIMPLEMENTED.
+                        ConPuts(StdErr, L"'t' option is FIXME (Accepted option though unimplemented feature).\n");
                         break;
                     case L'v':
-                        // FIXME!
-                        ConPuts(StdOut, L"got v\n");
+                        // UNIMPLEMENTED.
+                        ConPuts(StdErr, L"'v' option is FIXME (Accepted option though unimplemented feature).\n");
```

Comment:
Why?

Reply:
(Why what?) Changed what looks like a dev WIP to a more user friendly result.

Reply:
This does not like a comment meant for a user....

Reply:
I think it is an improvement over current code.
Feel free to suggest something even better...